<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ForgotPassword</class>
 <widget class="QDialog" name="ForgotPassword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Forgot Password</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <widget class="QPushButton" name="pbGetQuestion">
     <property name="text">
      <string>Get Question</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lSecurityQ">
     <property name="text">
      <string>Security Question:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lSecurityAns">
     <property name="text">
      <string>Enter Security Answer</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="leSecurityAns"/>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="leNewPass"/>
   </item>
   <item row="9" column="1">
    <widget class="QLineEdit" name="leNewPassConfirm"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lUsername">
     <property name="text">
      <string>Enter your Username:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="lNewPassConf">
     <property name="text">
      <string>Confirm New Password</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="lNewPass">
     <property name="text">
      <string>New Password</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="leUsername"/>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLabel" name="lSecurityQFromServer">
     <property name="text">
      <string>Submit Username!</string>
     </property>
    </widget>
   </item>
   <item row="10" column="2">
    <widget class="QPushButton" name="pbChangePass">
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="leDob"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lDob">
     <property name="text">
      <string>DOB: dd/mm/yyyy</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
